tests/src/dir.common.props

```diff
@@ -40,7 +40,7 @@
     <BaseOutputPath Condition="'$(__TestRootDir)' != ''">$(__TestRootDir)\</BaseOutputPath>
     <BaseOutputPathWithConfig>$(BaseOutputPath)\$(OSPlatformConfig)\</BaseOutputPathWithConfig>
     <BinDir>$(BaseOutputPathWithConfig)</BinDir>
-    <BaseIntermediateOutputPath>$(ProjectDir)\..\bin\tests\obj\$(OSPlatformConfig)\Managed</BaseIntermediateOutputPath>
+    <BaseIntermediateOutputPath>$(ProjectDir)\..\bin\tests\obj\$(OSPlatformConfig)\Managed\</BaseIntermediateOutputPath>
```

Comment:
It looks like the convention used to be no trailing slashes, and now they're included, so maybe `\`'s after `$(ProjectDir)` and such should be removed? They aren't inserted after `$(ProjectDir)` in tests/dir.props for example.